x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -554,14 +554,13 @@ export class Plugin implements ISecuritySolutionPlugin {
       APP_UI_ID,
       getAssistantTools(config.experimentalFeatures.assistantNaturalLanguageESQLTool)
     );
-    plugins.elasticAssistant.registerFeatures(APP_UI_ID, {
+    const features = {
       assistantBedrockChat: config.experimentalFeatures.assistantBedrockChat,
       assistantKnowledgeBaseByDefault: config.experimentalFeatures.assistantKnowledgeBaseByDefault,
       assistantModelEvaluation: config.experimentalFeatures.assistantModelEvaluation,
-    });
-    plugins.elasticAssistant.registerFeatures('management', {
-      assistantModelEvaluation: config.experimentalFeatures.assistantModelEvaluation,
-    });
+    };
+    plugins.elasticAssistant.registerFeatures(APP_UI_ID, features);
+    plugins.elasticAssistant.registerFeatures('management', features);
```

Comment:
The `management` plugin should have all the same features as `securitySolution`, so bundling that way they're not missed when the next one is added.